deployments/helm/KubeArmorOperator/values.yaml

```diff
@@ -4,3 +4,26 @@ kubearmorOperator:
     repository: kubearmor/kubearmor-operator
     tag: latest
   imagePullPolicy: IfNotPresent
+
+  configSpec:
+    defaultCapabilitiesPosture: audit
+    defaultFilePosture: audit
+    defaultNetworkPosture: audit
+
+    defaultVisibility: process,file,network
+
+    kubearmorImage:
+        image: kubearmor/kubearmor:stable
+        imagePullPolicy: Always
+
+    kubearmorInitImage:
+        image: kubearmor/kubearmor-init:stable
+        imagePullPolicy: Always
```

Comment:
Shall we remove use of stable tag altogether and start using v0.11.0 tag?
We can remove dependency on `STABLE_RELEASE` altogther? WDYT? CC: @daemon1024

Reply:
Then won't we have to update the charts every time we create a release?